src/Rootfs.jl

```diff
@@ -1100,3 +1107,83 @@ function installed_shards()
 
     return all_compiler_shards()[idx]
 end
+
+"""
+    manage_shards(; sort_by=:name, rev=false)
+
+Open a prompt allowing a user to selectively remove downloaded compiler shards.
+By default, the shards are sorted by name, alternatively you can sort them by
+file size on disk by specifying `sort_by=:size`. With `rev=true` you can
+reverse the sort order.
+"""
+function manage_shards(; sort_by=:name, rev=false)
+    # Get all installed shards
+    shards = installed_shards()
+
+    # Obtain directory sizes for the artifacts
+    totalsizes = totalsize.(Artifacts.artifact_path.(shard_source_artifact_hash.(shards)))
+
+    # Sort shards and totalsizes
+    if sort_by === :name
+        perm = sortperm(shards, rev=rev)
+    elseif sort_by == :size
+        perm = sortperm(totalsizes, rev=rev)
```

Comment:
My simple rule of thumb is: ***ALWAYS*** start keyword arguments with `;`, and then you don't need to care about the few corner cases where the comma wouldn't work
```suggestion
        perm = sortperm(shards; rev)
    elseif sort_by == :size
        perm = sortperm(totalsizes; rev)
```

Reply:
Thanks, makes total sense!